docs/CONTRIBUTING.md

```diff
@@ -72,45 +72,59 @@ sign for Google or Facebook open-source projects.
 
 ### Core
 
-* `anim` - core animation controller.
 * `api` - external API of the application.
 * `base` - some base things, like macros, logging, caches etc.
 * `coding` - I/O classes and data processing.
 * `drape` - the new graphics library core.
 * `drape_frontend` - scene and resource manager for the Drape library.
-* `drape_head` - an application that uses the Drape library.
 * `generator` - map building tool.
+* `geocoder` - geocoding process.
 * `geometry` - geometry primitives we use.
-* `graphics` - the current graphics library's core
-* `gui` - right, the GUI.
 * `indexer` - processor for map files, classificator, styles.
 * `map` - app business logic, including a scene manager.
 * `platform` - platform abstraction classes: file paths, http requests, location services.
 * `routing` - in-app routing engine.
+* `routing_common` - models and conversions used for routing.
 * `search` - ranking and searching classes.
-* `sound` - text-to-speech functions.
 * `std` - standard headers wrappers, for Boost, STL, C-rt.
 * `storage` - map reading function.
+* `tracking` - tracking of user's location.
+* `traffic` - real-time traffic information.
+* `transit` - routing for public transport.
+* `ugc` - user generated content, such as reviews.
 
 ### Other
 
 Some of these contain their own README files.
 
 * `3party` - external libraries, sometimes modified.
 * `android` - Android UI.
+* `cmake` - CMake helper files.
 * `data` - data files for the application: maps, styles, country borders.
 * `debian` - package sources for Debian.
+* `descriptions` - descriptions of features in different languages.
+* `editor` - editor of resources.
+* `feature_list` -
 * `installer` - long-abandoned installer for Windows.
-* `integration_tests` - routing tests for map files.
 * `iphone` - iOS UI.
-* `pedestrian_routing_benchmarks` - said benchmarks.
+* `kml` - manipulation of KML files.
+* `local_ads` - featured places on the map, for example shops.
+* `mapshot` - generate screenshots of maps, specified by coordinates and zoom level.
+* `metrics` - measuring user metrics.
+* `openlr` - dynamic location referencing (cross referencing maps).
+* `partners_api` - API for partners of the MAPS.ME project.
+* `pyhelpers` - some helper functions for python code.
 * `qt` - desktop application.
 * `qt_tstfrm` - widgets for visual testing.
+* `shaders` - shaders for rendering.
 * `skin_generator` - a console app for building skin files with icons and symbols.
+* `software_renderer` - renderer of the software
```

Comment:
this is wrong

Reply:
For the descriptions that are currently missing on the master, wouldn't it require a high priority to fix them? As a new programmer, I have no idea what some packages in the project are. This file could help remove a lot of confusion. Especially as not all code is commented.

Also, `renderer of the software` may not be very descriptive but if it's wrong, it makes me wonder if the name `software_renderer` is right. Should we refactor and rename this component? Or do you mean it is wrong because it is not very descriptive?